plugins/modules/project.py

```diff
@@ -47,6 +47,10 @@
       type: str
       aliases:
         - organization
+    scm_branch:
+      description:
+        - The scm branch of the git project.
+      type: str
```

Comment:
Can you please add `version_added: 2.4.0`?

Reply:
added

Reply:
integration test added , the target branch is stable-1.1